fractal-puppy/Engine
# Feature/flying skull enemy

The new Enemy the flying skull is now free and will never cease to stalk you wherever you go!

To test:
Load EnemiesTestFlyingSkull and check that it behaves correctly. (make sure the values for the script make sense, attack duration and flee duration i.e.)
If you have any troubles compiling or loading make sure to delete library and recompile scripts.

Expected behaviour can be found here:
https://docs.google.com/document/d/1WvzfhFyV8ZBwrPBzLWg_CHSwM3yT7aH4C0j4k7aAy3Y/edit
